client/app/queue/AddCavcRemandView.jsx

```diff
@@ -71,6 +72,19 @@ const AddCavcRemandView = (props) => {
   const [issues, setIssues] = useState({});
   const [instructions, setInstructions] = useState(null);
 
+  const supportedDecisionTypes = {
+    [CAVC_DECISION_TYPES.remand]: featureToggles.cavc_remand,
+    [CAVC_DECISION_TYPES.straight_reversal]: featureToggles.reversal_cavc_remand,
+    [CAVC_DECISION_TYPES.death_dismissal]: featureToggles.dismissal_cavc_remand
+  };
+  const supportedRemandTypes = {
+    [CAVC_REMAND_SUBTYPES.jmr]: featureToggles.cavc_remand,
+    [CAVC_REMAND_SUBTYPES.jmpr]: featureToggles.cavc_remand,
+    [CAVC_REMAND_SUBTYPES.mdr]: featureToggles.mdr_cavc_remand
+  };
+  const filteredDecisionTypes = typeOptions.filter((typeOption) => supportedDecisionTypes[typeOption.value]);
+  const filteredRemandTypes = subTypeOptions.filter((subTypeOption) => supportedRemandTypes[subTypeOption.value]);
```

Comment:
Filter out options that do not have a corresponding feature toggle toggled on

Reply:
Nice! Also might consider adding your comment to the file itself
```suggestion
// filter out options that do not have a corresponding feature toggle toggled on
  const filteredRemandTypes = subTypeOptions.filter((subTypeOption) => supportedRemandTypes[subTypeOption.value]);
```